src/pages/workspaces/workspace/Data.js

```diff
@@ -1002,6 +1047,26 @@ const WorkspaceData = _.flow(
               }
             }, [`${type} (${typeDetails.count})`])
           }, sortedEntityPairs),
+          div({ style: Style.navList.heading }, ['Snapshots']),
+          _.map(([snapshotName, { id: snapshotId, entityMetadata: snapshotTables }]) => {
+            const snapshotTablePairs = _.flow(_.toPairs, _.sortBy(_.first))(snapshotTables)
+            return h(SnapshotCollapse, {
+              buttonStyle: { color: colors.dark(), fontWeight: 600 },
+              style: { fontSize: 14, marginRight: '0.5rem', lineHeight: '50px', padding: '0 1.5rem', borderBottom: `1px solid ${colors.dark(0.2)}` },
+              title: snapshotName
+            }, [
+              p({ style: { fontSize: 14, lineHeight: '1.5' } },
+                _.map(([tableName, { count }]) => {
```

Comment:
@panentheos What are your thoughts on having a map as the children array, rather than within it? I think we try to avoid that, right? Either keying it or putting it as the contents of a Fragment?

Reply:
It's not the end of the world, but we should avoid dynamically generating the children array because it prevents React from optimizing rendering in some cases. Here, we should add an explicit array to the `div` above, such that this `_.map` expression is contained within it. And because we add a `key` to each `DataTypeButton`, it should work without warnings.